tools/breeze/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "breeze"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Use workspace package info like the other crates? This was recently merged.

Reply:
Breeze is not part of the workspace. So Im not able to use this.